.gitlab-ci.yml

```diff
@@ -89,3 +92,19 @@ deploy_to_reliability_env:
     UPSTREAM_PACKAGE_JOB: build
     UPSTREAM_PROJECT_NAME: $CI_PROJECT_NAME
     FORCE_TRIGGER: $DEPLOY_TO_REL_ENV
+
+tracer-base-image:
```

Comment:
So IIUC, the aim is to build an image that will be used by the rel env pipeline when we trigger it in `deploy_to_reliability_env`, right? So, why don't we add the code in the previous step? 

More generally, I believe our Azure devops pipeline should own this. So I believe we should: 
On each merge to master:
- Trigger a GH action from the Azure Devops pipeline
- Add the new version on GHCR from there

Which is BTW almost what Zach has done in this PR: https://github.com/DataDog/dd-trace-dotnet/pull/3769 where we push our container init image to GHCR. So, Ideally, I'd like to follow that pattern.

Reply:
Thanks @pierotibou . I have made the changes as you indicated, but I am not very familiar with Azure...how could I test it before merge?